chrome/locale/en-US/zotero/zotero.ftl

```diff
@@ -221,9 +221,11 @@ rtfScan-output-file = Output File
 
 rtfScan-no-file-selected = No file selected
 rtfScan-choose-input-file =
-    .label = Choose Input File...
+    .label = Choose File…
+    .aria-label = Choose Input File
 rtfScan-choose-output-file =
-    .label = Choose Output File...
+    .label = Choose File…
```

Comment:
This should use a `general-choose-file` variable so it doesn't have to be localized twice.

Reply:
Right, I moved this into `general-choose-file = Choose File…`